src/import/download_objectstore_files.py

```diff
@@ -26,9 +26,9 @@
 )
 
 EXPECTED_FILES = [
```

Comment:
Variable holds `file, age` array. Name doesn't properly reflect that. Maybe `EXPECTED_FILE_AGE_MAPPING`

Reply:
Ik heb het veranderd naar EXPECTED_FILES_AND_MAX_AGE